<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Playfair_cipher</class>
 <widget class="QMainWindow" name="Playfair_cipher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(84, 84, 84);
color: rgb(255, 255, 255);
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Zasifrovat_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(206, 206, 206);
background-color: rgb(0, 148, 202);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Zašifrovat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Desifrovat_button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>270</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(206, 206, 206);
background-color: rgb(0, 148, 202);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Dešifrovat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mezivysledky">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(206, 206, 206);
background-color: rgb(0, 148, 202);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Mezivýsledky</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Zobrazeni_tabulky">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>500</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(206, 206, 206);
background-color: rgb(0, 148, 202);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Zobrazit tabulku</string>
    </property>
   </widget>
   <widget class="QLabel" name="Nadpis_zasifrovat">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(255, 255, 255);
color: rgb(73, 255, 79);</string>
    </property>
    <property name="text">
     <string>Zašifrovat</string>
    </property>
   </widget>
   <widget class="QLabel" name="Nadpis_desifrovat">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(255, 255, 255);
color: rgb(73, 255, 79);</string>
    </property>
    <property name="text">
     <string>Dešifrovat</string>
    </property>
   </widget>
   <widget class="QLabel" name="Nadpis_vlozit_text_sifrovani">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Vložte text pro zašifrování:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Nadpis_vlozit_text_desifrovani">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Vložte text pro dešifrování: </string>
    </property>
   </widget>
   <widget class="QLabel" name="Vlozt_klic_sifrovani">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Klíč:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Vlozit_klic_desifrovani">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Klíč: </string>
    </property>
   </widget>
   <widget class="QLabel" name="Sifrovany_text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Zašifrovaný text:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Desifrovany_text">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>340</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Dešifrovaný text: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Klic_k_sifrovani">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Klic_k_desifrovani">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>230</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Nadpis">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Playfairova šifra</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Text_pro_sifrovani">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Text_pro_desifrovani">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>130</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Zasifrovany_text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Desifrovany_text_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>380</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="matice_vypis">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="mezivysledky_vypis">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>211</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="key_size">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlayfairova_ifra">
    <property name="title">
     <string>Playfair Cipher</string>
    </property>
   </widget>
   <addaction name="menuPlayfairova_ifra"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
